### DISCUSSION TITLE
购买专业版后，无法收到恢复码？怎么办？

### DISCUSSION THREAD

**Discussion by banjinANDbaliang** [getlantern/lantern-forum-cn#669] [Q&A]
购买专业版后，无法收到恢复码？怎么办？

**Comment by MaxMKMax** [comment]
@banjinANDbaliang 您好. 请您放心, 我们会帮您解决问题! 
为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Reply by banjinANDbaliang** [reply]
你好，已经提交了

**Reply by banjinANDbaliang** [reply]
@Mike777Khoo
